helm/chart/router/values.yaml

```diff
@@ -43,10 +46,12 @@ serviceAccount:
 
 podAnnotations: {}
 
-podSecurityContext: {}
+podSecurityContext:
+  {}
```

Comment:
NIT: Restore these (and the subsequent changes) to same line to maintain consistency and reduce change noise.

Reply:
**Curse you, ~Perry the Platypus~`"editor.formatOnSave": true`!**